src/oc/web/actions/activity.cljs

```diff
@@ -239,13 +242,30 @@
       (dis/dispatch! [:modal-editing-activate]))
     (dis/dispatch! [:modal-editing-deactivate])))
 
+
+(declare entry-save)
+
 (defn entry-save-on-exit
-  [edit-key activity-data entry-body]
+  [edit-key activity-data entry-body section-editing]
   (let [entry-map (assoc activity-data :body entry-body)
         cache-key (get-entry-cache-key (:uuid activity-data))]
     (uc/set-item cache-key entry-map
      (fn [err]
        (when-not err
+         (when (and (not= "published" (:status activity-data))
+                    (:has-changes activity-data)
+                    (not (:auto-saving activity-data)))
+           ;; dispatch that you are auto saving
+           (dis/dispatch! [:update [:entry-editing]
+                           #(merge % activity-data {:auto-saving true})])
```

Comment:
You need to use `edit-key` here instead of `:entry-editing` or you'll always do it only for the `entry-edit` component.

Also much simpler to use:
`(dis/dispatch! [:input [:entry-editing :auto-saving] true])`
sorry to bother on these :input and :update but i am trying to make more clear how they works since they are simple but confusing.

Reply:
So I used the merge function because I need the links information and the revision information if the entry is auto saved.  Using the suggested input dispatch doesn't add that information.